webpack.config.js

```diff
@@ -63,6 +63,9 @@ const baseConfig = {
       loader: 'file-loader',
     }],
   },
+  node: {
+    process: false,
```

Comment:
Webpack comes with a polyfill for the "process" api it includes by default. I found it while working with the tests.

This wouldn't be a big issue if we didn't use the version built for the browser (so including the polyfill) for the worker in tests...

One bad thing it does is process.env is empty.

But we have further issues with our setup: because we build with NODE_ENV=development, and replace process.env.NODE_ENV with the value while building, the worker logs all state changes in tests.

So I don't think this change actually changes anything besides removing the polyfill for the browser build -- which we don't need.